prow/config.yaml

```diff
@@ -2805,6 +2805,79 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0/docker-graph
 
+  - name: pull-kubernetes-kubemark-e2e-gce-scale
+    agent: kubernetes
+    always_run: false
+    max_concurrency: 1
+    skip_branches:
+    - release-1.6  # not supported
+    - release-1.7  # per-release image
+    - release-1.8  # per-release image
```

Comment:
nit: 1.9 as well? or you want to do `branches: master` instead?

Reply:
Done.. thanks!